android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/AndroidPayCheckoutSession.java

```diff
@@ -15,22 +17,45 @@ public AndroidPayCheckoutSession() {
         this.logger = new AndroidLogger();
     }
 
-    public AndroidPayCheckoutSession(ILogger logger) {
+    AndroidPayCheckoutSession(ILogger logger) {
         this.logger = logger;
     }
 
+    //CHECKSTYLE:OFF
```

Comment:
Checkstyle doesn't like that we have more than 7 method parameters but turning this into a JSON object instead just to get parameters from Unity doesn't seem necessary.